pages/item-new/_.vue

```diff
@@ -0,0 +1,369 @@
+<template>
+  <div>
+    <NotificationBanner
+      v-if="redirectNotificationsEnabled"
+      :notification-url="notificationUrl"
+      :notification-text="$t('linksToClassic.record.text')"
+      :notification-link-text="$t('linksToClassic.record.linkText')"
+    />
+    <b-container v-if="error">
+      <AlertMessage
+        :error="error"
+      />
+    </b-container>
+    <template
+      v-else
+      data-qa="item page"
+    >
+      <b-container
+        fluid
+        class="bg-white mb-3"
+      >
+        <!-- TODO: add swiper here
+          Swiper also includes download, share, right statement
+          Potential useful data:
+
+          :url="selectedMedia.about"
+          :europeana-identifier="identifier"
+          :use-proxy="useProxy"
+          :rights-statement="rightsStatement"
+          :data-provider-name="dataProvider.values[0]"
+          :data-provider-lang="dataProvider.code"
+          :is-shown-at="isShownAt"
+
+          Keep/reuse client-only?
+      -->
+        <client-only />
+      </b-container>
+      <b-container>
+        <b-row class="mb-3">
+          <b-col>
+            <!-- TODO: add new title + description box
+              Useful data:
+
+              titlesInCurrentLanguage
+              descriptionInCurrentLanguage
+            -->
+          </b-col>
+        </b-row>
+        <b-row class="mb-3">
+          <b-col>
+            <h2
+              v-if="relatedEntities && relatedEntities.length > 0"
+              class="related-heading text-uppercase"
+            >
+              {{ $t('contentYouMightLike') }}
+            </h2>
+            <!-- TODO: related content
+              Useful data:
+
+              :entities="relatedEntities"
+              data-qa="related entities"
+            -->
+          </b-col>
+        </b-row>
+        <b-row class="mb-3">
+          <b-col>
+            <MetadataBox
```

Comment:
The metadatbox will be getting rounded corners again.

Reply:
Ah, I have missed that memo. I have added some temp styling and a todo.